File: PHP/aksesoris.php
<?php
    require_once "petshop.php";

    // ATRIBUT PRIVATE
    class aksesoris extends petshop{
        private $jenis = '';
        private $bahan = '';
        private $warna = '';

    // CONSTRUCTOR
    public function __construct($id, $nama_produk, $harga_produk, $stok_produk, $foto, $jenis, $bahan, $warna) {
        parent::__construct($id, $nama_produk, $harga_produk, $stok_produk, $foto);
        $this->jenis = $jenis;
        $this->bahan = $bahan;
        $this->warna = $warna;
    }

    // GETTER
    public function get_jenis(){
        return $this->jenis;
    }
    public function get_bahan(){
        return $this->bahan;
    }
    public function get_warna(){
        return $this->warna;
    }   
    
    // SETTER
    public function set_jenis($jenis){
        $this->jenis = $jenis;
    }
    public function set_bahan($bahan){
        $this->bahan = $bahan;
    }
    public function set_warna($warna){
        $this->warna = $warna;
    }
}
?>

File: CPP/aksesoris.cpp
#include <bits/stdc++.h>
#include <iostream>
#include <string>
#include "petshop.cpp"

using namespace std;

class aksesoris : public petshop
{
    private:
        string jenis;
        string bahan;
        string warna;
    
    // CONSTRUCTOR
    public:
        aksesoris(){
            this->jenis = "";
            this->bahan = "";
            this->warna = "";
        }

        aksesoris(string id, string nama_produk, int harga_produk, int stok_produk, string jenis, string bahan, string warna) 
        : petshop(id, nama_produk, harga_produk, stok_produk)
        {
            this->jenis = jenis;
            this->bahan = bahan;
            this->warna = warna;
        }

        // DESTRUCTOR
        ~aksesoris(){
        }

    // GETTER
    string get_jenis(){
        return jenis; 
    }
    string get_bahan(){
        return bahan;
    }
    string get_warna(){
        return warna;
    }

    // SETTER
    void set_jenis(string jenis){
        this->jenis = jenis;
    }
    void set_bahan(string bahan){
        this->bahan = bahan;
    }
    void set_warna(string warna){
        this->warna = warna;
    }
};
